protocol/messenger_installations_test.go

```diff
@@ -162,7 +163,8 @@ func (s *MessengerInstallationSuite) TestSyncInstallation() {
 	s.Require().NoError(err)
 
 	// Create group chat
-	response, err := s.m.CreateGroupChatWithMembers(context.Background(), "group", []string{})
+	members := []string{common.PubkeyToHex(&contactKey.PublicKey)}
+	response, err := s.m.CreateGroupChatWithMembers(context.Background(), "group", members)
```

Comment:
Could you please shed a light into how does it improve the test?

Reply:
just to ensure we don't perform `validateAddedGroupMembers` while handling chat installation